catalog/ui/catalog-ui-metacard-sharing-impl/src/main/java/org/codice/ddf/catalog/ui/sharing/ShareableMetacardImpl.java

```diff
@@ -55,30 +52,27 @@ protected ShareableMetacardImpl(Metacard metacard) {
   }
 
   @VisibleForTesting
-  ShareableMetacardImpl() {
+  public ShareableMetacardImpl() {
     super(
         new MetacardTypeImpl(
             "metacard.sharing", Arrays.asList(new CoreAttributes(), new SecurityAttributes())));
   }
 
   @VisibleForTesting
-  ShareableMetacardImpl(String id) {
+  public ShareableMetacardImpl(String id) {
     this();
     setId(id);
   }
```

Comment:
❗️ It doesn't make sense to annotate `public` members as `VisibleForTesting`. Can we either remove the annotation or keep them hidden? I think we're supposed to use the static factory methods below. If that's the case we should also add javadoc to explain.